.travis.yml

```diff
@@ -1,21 +1,26 @@
+language: python
 sudo: false
 cache: pip
 
+python:
+- "2.7.13"
+
 addons:
+  postgresql: "9.5"
   apt:
     packages:
       - expect-dev  # provides unbuffer utility
       - python-lxml # because pip installation is slow
 
-language: python
-
-python:
-  - "2.7"
-
 env:
-  - VERSION="10.0" LINT_CHECK="1"
-  - VERSION="10.0" ODOO_REPO="odoo/odoo" LINT_CHECK="0"
-  - VERSION="10.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0"
+  global:
+  - VERSION="10.0" TESTS="0" LINT_CHECK="0" MAKEPOT="0"
+  - PHANTOMJS_VERSION="latest"
+
+  matrix:
+  - LINT_CHECK="1"
+  - TESTS="1" ODOO_REPO="odoo/odoo" MAKEPOT="1"
+  - TESTS="1" ODOO_REPO="OCA/OCB"
 
 virtualenv:
   system_site_packages: true
```

Comment:
Remove 25 and 26 lines since that py2.7.13 doesn’t have a virtualenv enabled 

(If we have this value from original template, then we have a mistake)